notebooks/cytotable-issue-75/cytotable_objgraph.py

```diff
@@ -0,0 +1,143 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py:light
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.15.0
+#   kernelspec:
+#     display_name: Python 3 (ipyflow)
+#     language: python
+#     name: ipyflow
+# ---
+
+# # CytoTable object graph analysis
+#
+# This notebook explores how CytoTable objects operate. The work is related to [CytoTable#75](https://github.com/cytomining/CytoTable/issues/75).
+
+# +
+import gc
+import sys
+
```

Comment:
```suggestion
import pathlib

```

Reply:
To avoid ImportError in this notebook - needs to be rerun as well

Reply:
Thank you @gwaybio ! Added fixes for this.